GliaWidgets/Sources/View/Chat/Message/Content/Text/ChatTextContentView.swift

```diff
@@ -89,19 +94,52 @@ class ChatTextContentView: BaseView {
     }
 
     private func setText(_ text: String?) {
-        if text == nil || text?.isEmpty == true {
+        guard let text, !text.isEmpty else {
             textView.removeFromSuperview()
-        } else {
-            if textView.superview == nil {
-                contentView.addSubview(textView)
-                textView.translatesAutoresizingMaskIntoConstraints = false
-                var constraints = [NSLayoutConstraint](); defer { constraints.activate() }
-                constraints += textView.layoutInSuperview(insets: kTextInsets)
-            }
-            textView.text = text
+            return
+        }
+
+        if textView.superview == nil {
+            contentView.addSubview(textView)
+            textView.translatesAutoresizingMaskIntoConstraints = false
+            var constraints = [NSLayoutConstraint](); defer { constraints.activate() }
+            constraints += textView.layoutInSuperview(insets: kTextInsets)
         }
+        textView.text = text
+
         textView.accessibilityIdentifier = text
     }
+
+    private func setAttributedText(_ text: NSAttributedString?) {
+        guard let text, !text.string.isEmpty else {
+            textView.removeFromSuperview()
+            return
+        }
+
+        if textView.superview == nil {
```

Comment:
And this also seems like a lot for this method. Don't know, seems a bit strange that setting the attributed text deals with adding/removing stuff from the superview, setting translatesAutoresizing..., etc.